packages/ethernaut-util/test/tasks/calldata.test.js

```diff
@@ -0,0 +1,365 @@
+const assert = require('assert')
+
+describe('calldata', function () {
+  let output
+  describe('when decoding 0x4054d7d608e735608ac6fe70609f9a9616215309e576e975414c4f73374b0b78', function () {
+    before('calldata', async function () {
+      hre.ethers.provider = new hre.ethers.JsonRpcProvider(
+        'https://ethereum-rpc.publicnode.com',
+      )
+      output = await hre.run(
+        {
+          scope: 'util',
+          task: 'calldata',
+        },
+        {
+          transactionId:
+            '0x4054d7d608e735608ac6fe70609f9a9616215309e576e975414c4f73374b0b78',
+        },
+        {
+          abi: [
```

Comment:
Use one of the abis in the filesystem instead of hardcoding it here for better readability?